packages/tsdocs/fixtures/monorepo/packages/pkg1/dist/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+Object.defineProperty(exports, '__esModule', {value: true});
+const pkg2_1 = require('pkg2');
```

Comment:
Is this file created by `tsc`? Do we need it for api-extractor? Isn't api-extractor working with TypeScript sources only? If we do need this files, then we should find a way how to build `dist` as part of the `npm run build` process, so that we don't have to commit them to git.

Reply:
I precompiled monorepo to reflect how `loopback-next` is set up. There is no need to rebuild it. I intensionally leave all files there to be consistent with our repo so that the acceptance test is run against a lerna repo with similar set of files.

Reply:
Fair enough.

Please capture this information in developer documentation. For example, you can create `packages/tsdocs/fixtures/monorepo/README.md` to explain what is in this fixture. Ideally, this documentation should also explain the process for updating the fixture files, should we need to do so in the future.